test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -1234,7 +1236,13 @@ describe("UserWallet - Lightning Pay", () => {
         // }))
         // await paymentOtherGaloyUser({walletPayee: userWalletB, walletPayer: userwalletC})
         expect(accountRecordA.contacts).toEqual(
-          expect.not.arrayContaining([expect.objectContaining({ id: usernameC })]),
+          expect.arrayContaining([expect.objectContaining({ id: usernameB })]),
+        )
+        expect(accountRecordB.contacts).toEqual(
```

Comment:
For this test, we may not be testing the new functionality completely since we aren't checking if the contact already existed on the user before we did this payment.

Ideally we should load each account before and test for the absence of the contact we're interested in, execute the payment, and then load the account again and confirm that the contact is now added.

Also if we take this approach, we may not be able to use this specific test since it runs twice (for each of the test cases) and so on the 2nd run the absence check would fail.

I'd recommend:
- adding the absence check as well before payment & presence check
- moving these contact checks to a new `it` test outside of the forEach loop